### ISSUE TITLE
VS Code C# link to Godot broken

### ISSUE THREAD

**Issue by kutoman** [godotengine/godot#42281]
**Godot version:**

v3.2.3.stable.mono.official

**OS/device including version:**

NVIDIA Quadro K2100M

**Issue description:**

Actually this is an issue I first reported in the VS Code Omnisharp issue section as it's about the C# VS Code extension but it was pointed out that this seems to be a "Godot sdk bug". Please check the description here: [omnisharp-vscode#4075](https://github.com/OmniSharp/omnisharp-vscode/issues/4075). What I've not mentioned there, is, that this began one day after I've upgraded to 3.2.3. Since then I can't use VS Code for Godot scripting anymore.

**Steps to reproduce:**
I've already tried different .NET versions but with no effect. Thus I don't know how to reproduce this issue.

**Minimal reproduction project:**

**Comment by Calinou** [comment]
cc @neikeq

**Comment by neikeq** [comment]
Can Godot build the project successfully?
Does the issue still happen after you built the project? (it may be because the nuget packages weren't restored)
If you've installed the Visual Studio Build Tools rather than the full Visual Studio IDE, make sure you've installed the .NET Core workload as described here: https://github.com/godotengine/godot/issues/41941#issuecomment-690490775 and https://github.com/godotengine/godot/issues/41941#issuecomment-690526211

**Comment by kutoman** [comment]
yes the mono builds via godot get completed successfully and the .NET Core workload is installed, but the issue still remains.

this here gets logged in the omnisharp log (in case it's relevant):
```
[info]: OmniSharp.MSBuild.Discovery.MSBuildLocator
        Located 3 MSBuild instance(s)
            1: Visual Studio Build Tools 2019 16.7.30517.126 - "C:\Program Files (x86)\Microsoft Visual Studio\2019\BuildTools\MSBuild\Current\Bin"
            2: Visual Studio Build Tools 2017 15.9.28307.1259 - "C:\Program Files (x86)\Microsoft Visual Studio\2017\BuildTools\MSBuild\15.0\Bin"
            3: StandAlone 16.8.0 - "c:\Users\User\.vscode\extensions\ms-dotnettools.csharp-1.23.2\.omnisharp\1.37.1\.msbuild\Current\Bin"
```

**Comment by passeride** [comment]
I got a similar issue that was resolved by changing the build tool, when I changed from msbuild to dotnet it behaved better (on Emacs LSP so essentially omnisharp).
Not sure if it helps, but worth a try friend :eyeglasses:

**Comment by kutoman** [comment]
@passeride thanks for your advice. The issue remains unfortunately despite of switching

**Comment by akien-mga** [comment]
Do you still reproduce this issue with 3.3 RC 6 or later?

**Comment by kutoman** [comment]
@akien-mga I've tested it with RC 7 and it works now!

**Comment by akien-mga** [comment]
Great news, thanks! Closing as fixed in 3.3.

**Comment by ShotgunFlamez** [comment]
@akien-mga what do you mean by "Do you still reproduce this issue with 3.3 RC 6 or later?"

**Comment by Calinou** [comment]
> @akien-mga what do you mean by "Do you still reproduce this issue with 3.3 RC 6 or later?"

It asks whether you can still reproduce the bug described in the first post with Godot 3.3RC6 or later. This includes the latest stable version available as of writing, Godot 3.3.2.
